### ISSUE TITLE
Filelist field type JS ERROR in IE

### ISSUE THREAD

**Issue by robbie2491** [bolt/bolt#6542]
I have recently noticed a JS bug in IE10/IE11 on a windows 7 platform. This error may also exist on later versions of IE but have not managed to test. 

This error occurs when trying to remove a saved or recently added file from a filelist field type.

To reproduce:
- Add filelist field to content type
- Add file or files to field
- Open console log (f12)
- Click 'x' to delete recently added file

Result:
- File will not be removed. (this also happens with saved files)
- Error will then be in console:

Error: Object doesn't support property or method 'closest'

- Bolt version: 3
- PHP version: 5.4.45
- Installed via TAR file

**Comment by bobdenotter** [comment]
🔮

You'll need to tell us more: 

 - When does this happen? 
 - What version of Bolt?
 - What version of PHP / webserver? 
 - How was it installed?

**Comment by robbie2491** [comment]
Amended above

**Comment by bobdenotter** [comment]
> Bolt version: 3

There's a huge difference between 3.0 and 3.3. Which one exactly? If it's not a recent one, try upgrading to Bolt 3.2.10.

**Comment by robbie2491** [comment]
Bolt versions i have seen this error in:

- 3.0.11
- 3.14
- 3.2.6
- 3.2.7
- 3.2.9

**Comment by CarsonF** [comment]
which version of IE?

**Comment by robbie2491** [comment]
IE10/IE11 but may also exist in edge 13/14 but have not been able to test on a windows 10 machine yet

**Comment by xiaohutai** [comment]
I'm seeing a
```
Object doesn't support property or method 'closest'
```
on statement:
```
c.target.closest(".item").remove()
```
I believe it may have to be
```
$(c.target).closest(".item").remove()
```

Compare the two `closest` functions:
https://github.com/bolt/bolt/blob/6307ce400eb9e74b7f6140bc7a8e684ce38b50c4/app/src/js/widgets/fields/fieldFilelist.js#L157-L166

**Comment by bobdenotter** [comment]
Thans, @robbie2491 and @xiaohutai. Fix is in the PR queue at #6697
